src/elements/autocomplete/autocomplete-base-element.ts

```diff
@@ -270,6 +270,13 @@ abstract class SbbAutocompleteBaseElement extends SbbNegativeMixin(
 
   private _handleSlotchange(): void {
     this._highlightOptions(this.triggerElement?.value);
+    /**
+     * In Angular, it's possible to filter out option with an opened panel on input change.
```

Comment:
As it is possible also without Angular, I suggest to remove this.
```suggestion
     * It's possible to filter out option with an opened panel on input change.
```